.env.yaml

```diff
@@ -0,0 +1,2 @@
+FOO: bar
```

Comment:
Can you please remove this file or put it in a folder called `contrib` and link to it?